docs/topics/developer-lightspeed/con_model-as-a-service.adoc

```diff
@@ -0,0 +1,23 @@
+:_newdoc-version: 2.15.0
+:_template-generated: 2024-2-21
+
+:_mod-docs-content-type: CONCEPT
+
+[id="model-as-a-service_{context}"]
+= Deploying an LLM as a scalable service 
+
+[role="_abstract"]
+
+The code suggestions differ based on various parameters about the large language model (LLM) used for an analysis. Therefore, model-as-a-service enables you more control over using {mta-dl-plugin} with an LLM that is trained for your specific requirements than general purpose models from the public AI providers.
+
+{mta-dl-plugin} is built to analyze better when it can access code changes resulting from analysis performed at scale across many application teams. In an enterprise, changes at scale become more consistent when the LLMs that generate the code change suggestions are shared across application teams than when each team uses a different LLM. This approach calls for a common strategy in an enterprise to manage the underlying resources that power the models that must be exposed to multiple members in different teams.
```

Comment:
> is built to analyze better when it can access code changes resulting from analysis performed at scale across many application teams

This isn't quite accurate, our analysis does not improve.  The analysis is same rules driven analysis.

Code suggestions may potentially improve by better/improved "context" from the Solution Server. 

So it's the fixing of a problem that may get better the more an organization uses MTA (if they use the Solution Server running).